packages/runtime-node/src/node-environment.ts

```diff
@@ -6,12 +6,13 @@ import {
     RuntimeEngine,
     FeatureLoadersRegistry,
     IPreloadModule,
+    Environment,
 } from '@wixc3/engine-core';
 import { init, remapToUserLibrary, clear } from './extrenal-request-mapper';
 
 import type { IEnvironment, StartEnvironmentOptions, IStaticFeatureDefinition } from './types';
 
-export async function runNodeEnvironment({
+export async function runNodeEnvironment<ENV extends Environment>({
```

Comment:
maybe in this case it's better to pass the environment and not the name, and to infer the env type inside runNodeEnvironment?